.github/workflows/pr-agent.yml

```
@@ -0,0 +1,26 @@
+name: PR-Agent
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+permissions:
+  issue-comments: write
```

actionlint (1.7.12)

[error] unknown permission scope "issue-comments". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)